### DISCUSSION TITLE
Question about translation

### DISCUSSION THREAD

**Discussion by detobel36** [padloc/padloc#541] [Q&A] [answered]
Hello,

I don't know if you can help me... I don't understand the differences between two sentences in the translation...
https://github.com/padloc/padloc/blob/main/packages/locale/res/translations/_template.json#L1875,L1879

> The app could not establish a connection with **our servers**, please check your internet connection or try again later!

And

> The app could not establish a connection with **the our servers**, please check your internet connection or try again later!

Is it a typo ?
The only difference is the "**the our servers**".  But it's the first time I see this.  If it exist in English, could you give me an example of the difference ?

**Comment by BrunoBernardino** [comment, marked as answer]
Hi @detobel36 thank you so much for reaching out and asking about this! It is indeed a typo and I've already fixed it.
